### PR TITLE
chore(export): update export options for the components in the library

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#233](https://github.com/unimelb/ffam-prototype/issues/233)

Update the index.js file to make the components exportable and update their stories showing how to use in raw html and in vue mode. 

- [x] ListingWrap
- [x] ListItem
- [x] ArticleWrap
- [x] BlockQuotation
- [x] ButtonIcon
- [x] GenericCard 
- [x] ContactList
- [x] ContentBlock
- [x] Dropdown
- [x] VideoEmbed
- [x] SoundcloudEmbed
- [x] FigureWrap
- [x] Notice
- [x] PageBreadcrumbs
- [x] PageHeader
- [x] PageHeaderMin
- [x] Pathfinder
- [x] CardImage
- [x] CardLink
- [x] SectionDivider
- [x] SectionTwoCol
- [x] SectionWrap
- [x] SplitSection
- [x] SplitSectionQuote
- [x] SidePanel
- [x] SidePanelNavItem
- [x] SocialList
- [x] StatsGroup
- [x] StatsItem
- [x] SublinkMenu
- [x] SublinkMenuItem
- [x] Welcome

The idea of this PR is to make sure that all the components in the `pattern-lib` are reusable, and the stories show both ways of usage ( Raw HTML and Vue component ).

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 4b30756359c8bb1a30c17c0245bbac37d43464f5

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5c8f0144b4b0b90008b041b5

File: components/section/stories/SectionTwo/Story3.vue [hunk 1]
```diff
@@ -5,14 +5,13 @@
...
-      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Animi ipsa neque nemo, nam veritatis quasi ea perspiciatis laudantium aliquid alias velit sed quam tenetur dolorum, nihil quo pariatur cum commodi?</p>
-      Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut numquam ex perspiciatis harum amet laudantium sint error vitae exercitationem sapiente a nisi debitis ipsum sed voluptates natus, esse aut modi.
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Animi ipsa neque nemo, nam veritatis quasi ea perspiciatis laudantium aliquid alias velit sed quam tenetur dolorum, nihil quo pariatur cum commodi?</p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut numquam ex perspiciatis harum amet laudantium sint error vitae exercitationem sapiente a nisi debitis ipsum sed voluptates natus, esse aut modi.
     </div>
     <div slot="side">
       <FigureWrap>
         <img
           src="https://dummyimage.com/300x200.jpg"
-          alt="">
+          alt>
```
Comment (meligatt) [inline components/section/stories/SectionTwo/Story3.vue]: @gepifanio maybe pass nothing instead if no alt is needed?

File: components/article-wrap/stories/ArticleCategories.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <ArticleWrap
     :date="['2017-04-03', '3 Apr. 2017']"
-    :contact="contact"
+    :contact="[contact]"
```
[the remark this answers is not in the document]
Comment (gepifanio) [reply components/article-wrap/stories/ArticleCategories.vue]: Should be something with the linting. I will figure out why this has been changed. Thanks for pick it up!

File: components/cards/stories/GenericCard/Story4.vue [hunk 3]
```diff
@@ -21,20 +19,22 @@
 
 <script>
 import GenericCard from '../../GenericCard.vue';
+import genericCardDoc from './generic-card-docs.md';
```
[the remark this answers is not in the document]
Comment (gepifanio) [reply components/cards/stories/GenericCard/Story4.vue]: 👍

**Pull request merged**